src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -0,0 +1,87 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void TryCopyTo()
+        {
+            int[] src = { 1, 2, 3 };
+            int[] dst = { 99, 100, 101 };
+
+            Span<int> srcSpan = new Span<int>(src);
```

Comment:
BTW: It would be fine to do the reference assembly and CoreCLR façade plumbing right away. Most of the APIs are in CoreCLR already, and it is easy to add baseline for the few APIs that are not there yet.